LINUX/packaging/netmap.spec

```diff
@@ -0,0 +1,54 @@
+Name:          netmap
+Version:       %{version}
+Release:       %{buildnumber}%{?dist}
+Summary:       netmap for Network Monitor
+Group:         Development/Tools
+License:       Dual BSD/GPL
+BuildRequires: kernel-devel = @KERNEL_VERSION@
```

Comment:
We should also add "kernel = @KERNEL_VERSION@", so the build will only work if the kernel and kernel-devel match versions.